tests/file_uploads/views.py

```diff
@@ -137,6 +139,13 @@ def file_upload_content_type_extra(request):
     return JsonResponse(params)
 
 
+def file_upload_stopped(request):
+    request.upload_handlers.insert(0, StopUploadHandler())
+    request.upload_handlers.pop(2)
+    request.FILES
```

Comment:
Can you add a comment? as suggested in https://github.com/django/django/pull/13350#discussion_r481040831